src/test/java/com/payline/payment/wechatpay/service/PartnerTransactionIdServiceTest.java

```diff
@@ -0,0 +1,98 @@
+package com.payline.payment.wechatpay.service;
+
+import com.payline.payment.wechatpay.MockUtils;
+import com.payline.payment.wechatpay.enumeration.PartnerTransactionIdOptions;
+import com.payline.payment.wechatpay.util.constant.ContractConfigurationKeys;
+import com.payline.pmapi.bean.payment.ContractProperty;
+import com.payline.pmapi.bean.payment.request.PaymentRequest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.MockitoAnnotations;
+import org.mockito.Spy;
+
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.doReturn;
+
+ class PartnerTransactionIdServiceTest {
+    @InjectMocks
+    @Spy
+    private PartnerTransactionIdService underTest ;
+
+    @BeforeEach
+    void setup() {
+        underTest = PartnerTransactionIdService.getInstance();
+        MockitoAnnotations.initMocks(this);
+    }
```

Comment:
Tu devrais pourvoir simplifier un peu ici l'initialisation :
```suggestion
    @Spy
    private PartnerTransactionIdService underTest = PartnerTransactionIdService.getInstance();
```

Reply:
done